server/container_create.go

```diff
@@ -350,7 +350,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	attempt := metadata.GetAttempt()
 	containerInfo, err := s.storage.CreateContainer(s.imageContext,
 		sb.name, sb.id,
-		image, "",
+		image, image,
```

Comment:
ideally I'd love to fix this and just pass one argument describing an image (either ID or name), @nalind is that possible? (in a follow up)